### ISSUE TITLE
[GeoNature] Alerte utilisateur sur échéance date d'accès aux données précises

### ISSUE THREAD

**Issue by helenechauvin** [cbn-alpin/silene-geonature#207]
Serait-il possible de mettre en place une alerte lorsque la date d’accès aux données précises arrive à échéance pour informer l’utilisateur ?

**Comment by jpm-cbna** [comment]
Oui, c'est envisageable. Mais cela nécessite des développements supplémentaires. Il faut donc préciser cette fonctionnalité puis prévoir un budget et un temps pour ce développement.

**Comment by helenechauvin** [comment]
Ok. Pour le moment et avec les évolutions d'accès aux données précises pour l'avenir, ce n'est donc pas prioritaire comme développement.
